kong/plugins/statsd/schema.lua

```diff
@@ -181,8 +188,7 @@ return {
                 },
               },
           }, },
-          { allow_status_codes = {
-            type = "array",
+          { allow_status_codes = { description = "List of status code ranges that are allowed to be logged in metrics.", type = "array",
```

Comment:
The style on the field descriptions on this plugin is a bit inconsistent:
* Using "The xxx that does yyy" formula: "The IP address ..."
* Just saying "Does yyy": "Determines what sort of event... "
* Just saying "xxx": "List of status code ..."

It seems like the first style ("The xxx that does yyy") is the most usual one. I propose using that style in all the fields.

Reply:
@Guaris Your call.